File: src/com/leetcode/Easy/HeightChecker.java
package com.leetcode.Easy;

import java.util.*;

// 1051. Height Checker
public class HeightChecker {
    public int heightChecker(int[] heights) {
        int[] h = heights.clone();
        Arrays.sort(h);
        int count = 0;
        for (int i = 0; i < h.length; i++) {
            if (h[i] != heights[i])
                count++;
        }
        return count;
    }
}

File: JavaScript/Easy/HeightChecker.js
/**
 * 1051. Height Checker
 * @param {number[]} heights
 * @return {number}
 */
const heightChecker = function (heights) {
  let sortedHeights = heights.slice();
  sortedHeights.sort((a, b) => a - b);
  let count = 0;
  for (let i = 0; i < heights.length; i++) {
    if (heights[i] != sortedHeights[i]) {
      count++;
    }
  }

  return count;
};